### DISCUSSION TITLE
how to modify this thing style

### DISCUSSION THREAD

**Discussion by WingDust** [gitkraken/vscode-gitlens#2310] [Q&A]
![image](https://user-images.githubusercontent.com/45911286/199222017-93192a14-6b0f-4b81-9e29-1063fc7d7440.png)

**Comment by d13** [comment]
Those come from VS Code, not GitLens. 

I believe those are the `editorGutter` colors, which you can find listed here: https://code.visualstudio.com/api/references/theme-color#editor-colors

![Screen Shot 2022-11-01 at 10 33 52 AM](https://user-images.githubusercontent.com/293282/199259013-5c5f3fd5-2b2e-4a34-9729-6cff8c2b6f63.png)

Customizing theme colors guide: https://code.visualstudio.com/docs/getstarted/themes#_customizing-a-color-theme

Rough example:

```json
{
  "workbench.colorCustomizations": {
    "editorGutter.modifiedBackground": "#ffffff"
  }
}
```
